AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -240,19 +240,27 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   ASDN::MutexLocker l(__instanceLock__);
   CGSize calculatedSize = constrainedSize;
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   // if a preferredFrameSize is set, call the superclass to return that instead of using the image size.
-  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO)
+  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
+#if DEBUG
+    NSLog(@"Using -[ASDisplayNde preferredFrameSize] is deprecated.");
```

Comment:
I don’t think we should use `LOG()`. We would like to have the log message in development under all circumstances. And `LOG` is currently disabled in development and production